apps/web/package.json

```diff
@@ -69,6 +69,7 @@
     "bcryptjs": "^2.4.3",
     "classnames": "^2.3.1",
     "dotenv-cli": "^6.0.0",
+    "entities": "^4.4.0",
```

Comment:
@G3root @keithwillcode This new dependency seems like it's not being used. Let's revert this change please 🙏🏽